onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -143,6 +143,32 @@ enum DataLayout {
   L_1230 = 1,
 };
 
+static Status GetAxes(ModelBuilder& model_builder, const NodeUnit& node_unit,
+                      std::vector<int32_t>& axes) {
+  // Squeeze/Unsqueeze opset 13 uses input1 as axes
+  if (node_unit.SinceVersion() > 12) {
+    // If axes is not supplied, return an empty axes as default to squeeze all
+    if (node_unit.Inputs().size() > 1) {
```

Comment:
ONNX spec doesn't list axes as optional so I would expect the model would fail to load if the input was missing, and it's not necessary to check that there is more than one input.

Reply:
Note that this is different to Squeeze where the axes input is optional.